console/console-init/ui/src/modules/iot-device-detail/components/ConnectionGateway/ConnectionGateway.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router";
+import {
+  Card,
+  Flex,
+  FlexItem,
+  CardBody,
+  Title,
+  Text,
+  TextVariants,
+  CardTitle
+} from "@patternfly/react-core";
+import { StyleSheet, css } from "aphrodite";
+export interface IConnectionGatewayProps {
+  deviceList?: any;
+  deviceGroup?: any;
+}
+const styles = StyleSheet.create({
+  card_body: {
+    marginBottom: 20
+  },
+  text_margin: {
+    marginLeft: 25
+  }
+});
+
+export const ConnectionGateway: React.FC<IConnectionGatewayProps> = ({
+  deviceList,
+  deviceGroup
+}) => {
+  const { projectname, namespace } = useParams();
+
+  return (
+    <Card className={css(styles.card_body)}>
+      <CardTitle>
+        <Title id="device-info-gateways-title" headingLevel="h1" size="2xl">
+          Connection Gateways
+        </Title>
+      </CardTitle>
+      {deviceList.length > 0 && (
+        <text className={css(styles.text_margin)}>Gateway Devices</text>
+      )}
+      <CardBody>
```

Comment:
wrap the whole card body in the logic used above to render the text. Also use PF Text component instead of text